### DISCUSSION TITLE
How to determine which download failed (notification badge on icon)?

### DISCUSSION THREAD

**Discussion by mikeybuhtig** [aclap-dev/video-downloadhelper#221] [Help and Q&A]
### Your question

Hi, All,

I'm using VDH 8.2.1.1 on Firefox 123.0. I often queue up a dozen or more videos to download, typically in a private window.

Occasionally, a download will fail and the VDH on the toolbar will have a badge with a count of failures. However, I can't figure out a way to determine which download failed. I searched through everything I could find in VDH and Firefox for a pointer to which download had failed.

I finally had to wait until all the other downloads had finished, then look in the download folder, where I found the failed download still had the *.mp4 and *.mp4.part files. The *.mp4 file had zero bytes and the *.mp4.part file's size was the number of bytes actually downloaded.

In previous VDH versions, the downloading default was to have the browser handle the it, and I could keep a tab or window for downloads (Ctrl+J). Any download which failed would have a little "reload" icon on the right, which I could click. Sometimes it resumed the download and other times, it would start it from the beginning, but either way, I would know immediately which download(s) failed.

Having VDH manage the downloads _appears_ to be faster than having the browser manage them, but it's a challenge for the occasional failure to figure out which download to resume/restart.

Am I overlooking something? What's the best way to learn quickly which download failed so I can take action? And is it possible to resume the download, or must I start over? ...thanks!

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\<my ID>",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
Sign your name onto the long list of people asking for this sort of activity logging in VDH.  It doesn't exist.  What you did is what you have to do.  In addition, there is no way to resume an interrupted download from the point of failure.  I asked Michel about that a couple of years ago & that was his response.  It's in the old Google Groups forum if you can find it.

**Reply by mikeybuhtig** [reply]
Thank you for the confirmation.
